packages/perseus/src/widgets/matcher/matcher.tsx

```diff
@@ -78,7 +84,7 @@ export class Matcher extends React.Component<Props, State> implements Widget {
         this.setState({rightHeight: height});
     };
 
-    getUserInput: () => PerseusMatcherUserInput = () => {
+    _getUserInputFromSortable: () => PerseusMatcherUserInput = () => {
```

Comment:
The naming of this threw me for a loop for a second, as I wasn't super clear what "sortable" was, but that's mostly because I didn't realize we even had a sub-component of sortable here.

No action required, just learning new things about Perseus every day